artscii/src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './app/App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```

Comment:
Do we need this for anything or could it be deleted?

Reply:
Prob could be deleted unless we care about measuring user activity at some point